magni_bringup/launch/base.launch

```diff
@@ -5,9 +5,18 @@
     or Robot Commander.
 -->
 <launch>
+    <!-- 
+        This parameter is used to specify what level of capabilites
+        the robot should have after this launch file.
 
-    <node name="move_basic" pkg="move_basic" type="move_basic" output="screen">
+        Possible values are 'core', 'teleop', and 'navigation'
 
-	<remap from="/scan" to="/fuckoffff"/>
```

Comment:
classy xD

Reply:
I was hoping nobody would notice that I accidentally committed and then removed that :P 

Anyhow that's the result of me losing my mind at move_basic detecting all walls and not moving at all for the umpteenth time that evening with a deadline of -1 hours left.